### ISSUE TITLE
Bug: WordPress - Error: Minified React error #31;

### ISSUE THREAD

**Issue by plittlefield** [facebook/react#20722]
On multiple web sites I now have this error:-

`Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Brendered%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Nc (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:72:61)
    at https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:81:433
    at T (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:95:472)
    at zj (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:230:156)
    at Th (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:152:223)
    at tj (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:152:152)
    at Te (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:146:151)
    at https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:61:68
    at unstable_runWithPriority (https://www.domain.com/wp-includes/js/dist/vendor/react.min.js?ver=16.13.1:25:260)
    at Da (https://www.domain.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:60:280)
`

...when using the built-in Gutenberg Page Editor.

Does this relate to your code that WordPress uses?

Regards,

Paul

**Comment by plittlefield** [comment]
OK, have found the solution for me.

I am using NginX and did not have quite the correct settings for WordPress.

`try_files $uri $uri/ /index.php$is_args$args;`

I have updated my Wiki and it is here:-

http://wiki.indie-it.com/wiki/WordPress#Error:_Minified_React_error_.2331

Regards,

Paul

**Comment by EXStevens** [comment]
Thank u so much. It works.

I used to use this:
```nginx
location /wp-json/ {
    try_files $uri $uri/ /index.php;
}
```
it just solved the editor 404 problem, but this one solves all.

**Comment by yachaka** [comment]
Thank you very much @plittlefield this also was my issue and it was driving me crazy.
The editor would crash only when I opened the panels and I noticed it was also crashing on any pages editor and even on a brand new wordpress fresh install. I tried everything now to realize it was a server config issue.

This was easy to spot after I knew by looking in the "Site Health" tab, and it was showing "REST API did not behave correctly"
This was though showed as very little issue... Not even some red colors or bigger font. Though it was making my whole editor crash
